quilt/tools/store.py

```diff
@@ -29,11 +29,13 @@ class and its subclasses abstract file formats, file naming and
     PACKAGE_FILE_EXT = '.json'
     BUILD_DIR = 'build'
     OBJ_DIR = 'objs'
+    TMP_OBJ_DIR = 'objs/tmp'
 
-    def __init__(self, startpath='.'):
-        self._start_dir = startpath
+    def __init__(self, store_dir='.'):
```

Comment:
Do you think the arg should be the `quilt_packages` directory itself, or its parent? The former seems more logical - but then do we enforce that it has the correct name?